test/unit/specs/components/governance/PageProposal.spec.js

```diff
@@ -224,6 +224,14 @@ describe(`PageProposal`, () => {
       wrapper.setProps({ proposalId: `5` })
       expect(wrapper.find(`#vote-btn`).exists()).toEqual(false)
     })
+
+    describe(`user hasn't logged in`, () => {
+      it(`redirects to session page`, async () => {
+        store.commit(`setSignIn`, false)
+        await wrapper.vm.onVote()
```

Comment:
I think this behavior is wrong, see https://github.com/cosmos/voyager/issues/1918

Reply:
updated